public/src/css/style_old.css

```diff
@@ -0,0 +1,445 @@
+:root {
```

Comment:
There's no need to keep `style_old.css`, if we need it we can just go back in the commit history.

Reply:
removed